src/data/Mesh.cpp

```
@@ -55,7 +56,7 @@ void odc::data::Mesh::initialize( odc::io::OptionParser i_options, int_pt x, int
                                   Grid_XYZ* an_yz_grid
 #endif
                                 ) {
-  real taumax, taumin;
+  real taumax = 0.0, taumin = 0.0;
```

Init with invalid values to avoid silent errors, e.g.,
```
real taumax = -std::numeric_limits< real >::max();
```